unreleased/add-host-tags-manually-on-serverless-environments.yaml

```diff
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: datadogexporter
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Add host metadata tags to metrics on serverless environments
+
+# One or more tracking issues related to the change
+issues: []
```

Comment:
What issue should I fill in?

Reply:
We are discussing at #12417 if you can use a PR here instead of an issue. For now, can you open an issue describing what your PR fixes and add the issue number here?

Reply:
https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/12486 links to this PR now.